it/utils_for_tests.go

```diff
@@ -260,13 +260,11 @@ func SetupApp(t *testing.T, bundle *smith_v1.Bundle, serviceCatalog, createBundl
 			PrometheusRegistry: prometheus.NewPedanticRegistry(),
 			Name:               "smith",
 			RestConfig:         config,
-			Namespace:          cfg.Namespace,
```

Comment:
Why did you remove it? And workers? This should look like this:
```go
apl := &ctrlApp.App{
 			Logger:             logger,
 			MainClient:         mainClient,
			PrometheusRegistry: prometheus.NewPedanticRegistry(),
			Name:               "smith",
			RestConfig:         config,
			GenericNamespacedControllerOptions: GenericNamespacedControllerOptions{
			        GenericControllerOptions: GenericControllerOptions{ Workers: 2 },
			        Namespace: cfg.Namespace,
			},
			Controllers: []ctrl.Constructor{
				&app.BundleControllerConstructor{
					ServiceCatalogSupport: serviceCatalog,
				},
			},
```
Have you ran integration tests? They are broken now most likely. Or are processing each others namespaces.